### PR TITLE
Issue #164 html output test runner

### LINKED ISSUE #164: Support for HTML output for test runner
The platform should support html output. We should do something similar to what is already done with feedback, but now, we should add an output format option to the test result.

Or better, let the configuration of feedback and output format be an attribute of the language object. That way, messages exchanged between runner and platform would require less fields.

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #164

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@faloi please review

**Comment by faloi** [comment]
Is the migration for :output_content_type missing?

File: spec/models/content_type_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe ContentType do
```
Comment (faloi) [inline spec/models/content_type_spec.rb]: What are you testing here?

**Comment by flbulgarelli** [comment]
@faloi No, it is not. It is in last commit

**Comment by flbulgarelli** [comment]
I am testing that the content type properly flags the strings as html safe for makdown, so that they can be rendered by tempalte engine later without doing any escapes.

**Comment by flbulgarelli** [comment]
@faloi  you where right, migration was missing. Fixed

**Pull request merged**